src/cunumeric/unary/unary_red_util.h

```diff
@@ -89,6 +91,10 @@ constexpr decltype(auto) op_dispatch(UnaryRedCode op_code, Functor f, Fnargs&&..
       return f.template operator()<UnaryRedCode::PROD>(std::forward<Fnargs>(args)...);
     case UnaryRedCode::SUM:
       return f.template operator()<UnaryRedCode::SUM>(std::forward<Fnargs>(args)...);
+    case UnaryRedCode::SUM_SQUARES:
```

Comment:
Actually with all these changes below, I think we can just roll `SUM_SQUARES` into `VARIANCE` (and just pass `mu=0` when we want to use pure "sum squares" mode).

Reply:
I think doing that might ignore some performance benefits. Perhaps @jjwilke might comment further?

Reply:
To be addressed in a future refactoring. Resolving this thread for now.